tools/build_defs/scala/scala.bzl

```diff
@@ -95,6 +108,11 @@ def _collect_jars(ctx):
       jars += target.java.transitive_runtime_deps
   return jars
 
+def _args_for_suites(suites):
+  args = ["-o"]
+  for suite in suites:
+    args.extend(["-s", suite])
```

Comment:
Thanks Orr, responded on the main thread. Would it make sense to get this PR in as a "basic" way of specifying tests and possibly add more runners in the future?